common/utils/src/main/resources/error/error-conditions.json

```
@@ -1198,6 +1198,12 @@
     ],
     "sqlState" : "23505"
   },
+  "DUPLICATE_LABEL" : {
```

There are a few `_EXISTS` conditions already (`FIELD_ALREADY_EXISTS`, `INDEX_ALREADY_EXISTS`, `LOCATION_ALREADY_EXISTS`, `PATH_ALREADY_EXISTS`) , and let's follow naming convention in `error-conditions.json`, and also make it consisitent with the error text:
```suggestion
  "LABEL_ALREADY_EXISTS" : {
```